Lesson10/src/Lesson10.java

```diff
@@ -0,0 +1,23 @@
+import store.CashDesk;
+import store.Consumer;
+import threads.MyThread;
+
+public class Lesson10 {
+    public static void main(String[] args) {
+//        System.out.println(Thread.currentThread().getName()+ " thread start");
+//        MyThread myThread = new MyThread();
+//        myThread.run();
+//        System.out.println(Thread.currentThread().getName()+ " thread stop");
+//        myThread.interrupt();
+        CashDesk cashDesk = new CashDesk();
```

Comment:
so all your customers will go to the same cashdesk and task says to implement the solution when multiple customers can be served at the same time on different cashdesks. So I think you want to pass no 1 cashdesk but "all cashdesks of the store" to the customer, and let him decide which one to take